### PR TITLE
Test

### PR DESCRIPTION
**Pull request by venkuppu-chn**
-----
[View rendered README.md](https://github.com/venkuppu-chn/alex-test-1/blob/main/README.md)

### CODE REVIEW

**Comment by venkuppu-chn** [comment]
Please check

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # alex-test
 Mental
 Handicapped
```
Comment (cortx-admin) [inline README.md:3]: Please check

Comment (cortx-admin) [reply README.md:3]: Need clarity

Comment (cortx-admin) [reply README.md:3]: Post 1

Comment (cortx-admin) [reply README.md:3]: 1

Comment (cortx-admin) [reply README.md:3]: 1

Comment (venkuppu-chn) [reply README.md:3]: Okay

Comment (cortx-admin) [reply README.md:3]: Okay

**Pull request merged**